### DISCUSSION TITLE
Overlapping UI elements on users profile

### DISCUSSION THREAD

**Discussion by ghostwriter** [community/community#78077] [Mobile]
When a user has a long url in their profile information, the mobile application renders overlapping ui elements.

<details>
<summary>ℹ️ Additional Debug Info</summary>

<img src="https://github.com/community/community/assets/9754361/debe4bdb-9d62-4d54-8d2a-4943163cf085" alt="drawing" width="300"/>

GitHub for iOS, Version 1.142.0 (117339261)
Device: iPhone 13 (iOS 16.x)
</details>

**Comment by iskae** [comment]
Hi @ghostwriter 👋 

Thank you for reaching out reporting this ❤️ 

We are currently working on a fix for this 🙇 Please stay tuned for our future releases 😊 

Cheers!

**Reply by ghostwriter** [reply]
Hi, 

Okay, Thank you.

**Comment by ghostwriter** [comment]
Fixed as of Version: 1.145.0 (120653769)

Thank you, everyone involved, for playing your part, fulfilling this request.
